ciongke/cmd/hook/githubagent.go

```diff
@@ -239,9 +239,11 @@ func (ga *GitHubAgent) trustedPullRequest(pr github.PullRequest) (bool, error) {
 
 func (ga *GitHubAgent) askToJoin(pr github.PullRequest) error {
 	commentTemplate := `
-Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply "ok to test".
+Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply with "ok to test" on its own line.
 
 Regular contributors should join the org to skip this step.
+
+While we transition away from the Jenkins GitHub PR Builder plugin, "ok to test" commenters will need to be on the admin list defined in [this file](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml).
```

Comment:
During the transition, the jobs triggered by ciongke don't require this but the other jobs do. I want to make it clear that it's temporary.